pkg/health/health_test.go

```diff
@@ -0,0 +1,52 @@
+package health
+
+import (
+	"testing"
+
+	. "github.com/anthonybishopric/gotcha"
+)
```

Comment:
Need tests for:

* Catalog is healthy but KV is not present
* Catalog is healthy but KV is not
* KV is healthy but Catalog is not
* KV is healthy but Catalog is not present at all

All the above cases should result in passing health checks from `pkg/health`